flycheck.el

```diff
@@ -9549,6 +9549,20 @@ ignored."
   :package-version '(flycheck . "28"))
 (make-variable-buffer-local 'flycheck-rust-binary-name)
 
+(flycheck-def-option-var flycheck-rust-features nil rust-cargo
+  "List of features to activate during build or check.
+
+The value of this variable is a list of strings denoting features
+that will be activated to build the target to check. Features will
+be passed to `cargo check --features=FEATURES'.
+
+This is optional value defaulting to empty list of features."
```

Comment:
You can remove this last line, since `M-x describe-variable` will give its value, and Flycheck will add a string saying it's an option automatically by virtue of `flycheck-def-option-var`.